howso/direct/client.py

```diff
@@ -442,6 +445,20 @@ def _initialize_trainee(self, trainee_id: str):
             # If tracing is enabled, log the trainee version
             self.execute(trainee_id, "get_trainee_version", {})
 
+    def _initialize_transactional_trainee(self, trainee_id: str):
+        # Create a temporary trainee and initialize it in the normal way, then clone it with transactional mode on.
+        tmp_id = str(uuid.uuid4())
+        self._initialize_trainee(tmp_id)
+        try:
+            cloned = self.amlg.clone_entity(tmp_id, trainee_id,
```

Comment:
Why are we not able to create a new empty trainee with transactional enabled?

Reply:
The thing we're loading here is _the_ `howso.caml` file, and we don't want that to be overwritten.  So the path here is to `self.amlg.load_entity('howso.caml')`, initialize it, and then clone it to the real file path in transactional mode.

The other thing that happens here that we noted is that there are a lot of steps in trainee initialization.  We don't want that to happen in transactional mode; we just want to write out the initialized trainee once.